language-design.md

```diff
@@ -79,4 +105,7 @@ you do so? If not, why not?
 
 **Response**
 
+    I think they're at odds with one another. The syntax of Quorum/how one would read and write it seems very very similar to AppleScript. 
+    I do think that natural language should be included in the design of DSLs however. I personally enjoy the use of natural language in languages like SQL, where it's not exactly a natural language, but by utilizing natural language replacements it can be readable and understood by beginners and novices.  
+
```

Comment:
I have never programmed using a natural language, but I am not sure if I would like it. Once I learn a language my brain starts thinking that way when I am reading and writing code. I think that maybe a programming language that could be translated into a natural language could be very helpful because you can read and write in the natural language, but if there is a weird bug that is hard to understand, you could switch over to the standard programming language to test it.